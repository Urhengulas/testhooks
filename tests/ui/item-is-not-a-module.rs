use testhooks::testhooks;

#[testhooks]
struct S;

fn main() {}
