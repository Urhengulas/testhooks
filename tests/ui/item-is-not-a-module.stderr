error: item is not a module

  = help: `#[testhooks]` can only be used on modules

 --> $DIR/item-is-not-a-module.rs:4:1
  |
4 | struct S;
  | ^^^^^^^^^
